pkg/docker/cockpit-atomic-storage

```diff
@@ -195,12 +180,17 @@ def get_driver_info():
         conf = open(ds_conf_file, "r").read()
     else:
         conf = ""
-    m = re.search("dm.thinpooldev=([^ ]*)", conf)
+    m = re.search("--storage-driver ([^ ]*)", conf)
```

Comment:
Shouldn't this catch `--storage-driver=foo` too? I. e. `--storage-driver[= ]*([^ ]*)` ? (This isn't as strict as it could be, but good enough IMHO)

Reply:
Yes, it should. I made the `[= ]+` so that we don't match --stroage-driver-extra=foo